app/src/main/java/org/mozilla/fenix/share/SaveToPDFItem.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.share
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * A save to PDF item.
+ * @param onClick event handler when the save to PDF item is clicked.
+ */
+@Composable
+fun SaveToPDFItem(
+    onClick: () -> Unit,
+) {
+    Row(
+        modifier = Modifier
+            .height(48.dp)
+            .fillMaxWidth()
+            .clickable(onClick = onClick),
+    ) {
+        Column {
+            Divider(
+                color = FirefoxTheme.colors.borderPrimary,
+                thickness = 1.dp,
+            )
+
+            Row(modifier = Modifier.padding(top = 16.dp)) {
+                Icon(
+                    modifier = Modifier.padding(start = 20.dp, end = 36.dp),
```

Comment:
When items are clickable, IMO using padding instead of spacers would improve accessibility. (be aware of the modifiers order to keep padding clickable.)

Reply:
It's the entire row that's clickable here though, not the individual elements. The padding in this instance is just on the `Icon`.

Reply:
Yeah, I was talking about the general "we lean towards" part, adding an instance where that would not be the best choice.